src/ontology/run.sh

```diff
@@ -45,7 +45,7 @@ if [ -n "$USE_SINGULARITY" ]; then
         -W $WORK_DIR \
         docker://obolibrary/$ODK_IMAGE:$ODK_TAG $TIMECMD "$@"
 else
-    docker run -v $VOLUME_BIND -w $WORK_DIR -e ROBOT_JAVA_ARGS="$ODK_JAVA_OPTS" -e JAVA_OPTS="$ODK_JAVA_OPTS" --rm -ti obolibrary/$ODK_IMAGE:$ODK_TAG $TIMECMD "$@"
+    docker run -v $VOLUME_BIND -w $WORK_DIR -e ROBOT_JAVA_ARGS="$ODK_JAVA_OPTS" -e JAVA_OPTS="$ODK_JAVA_OPTS" -ti obolibrary/$ODK_IMAGE:$ODK_TAG $TIMECMD "$@"
```

Comment:
```suggestion
    docker run -v $VOLUME_BIND -w $WORK_DIR -e ROBOT_JAVA_ARGS="$ODK_JAVA_OPTS" -e JAVA_OPTS="$ODK_JAVA_OPTS" --rm -ti obolibrary/$ODK_IMAGE:$ODK_TAG $TIMECMD "$@"
```

Reply:
No changes to run.sh on main branch please.

Reply:
Oh, ok. We made this change together. Thought this was something we were changing permanently in order to address the issue of Python packages not being updated within the container between runs. It didn't work anyway :(, was gonna bring it up at next check-in meeting. It's still on my list of things to discuss whenever we have time.